StripePaymentSheet/StripePaymentSheet/Source/Helpers/PaymentSheetLinkAccount.swift

```diff
@@ -444,13 +438,13 @@ private extension PaymentSheetLinkAccount {
                     self?.currentSession = session.consumerSession
                     self?.publishableKey = session.publishableKey
                     completion(.success(()))
-                case .notFound(let errorMessage):
+                case .notFound:
                     completion(
-                        .failure(PaymentSheetError.unknown(debugDescription: errorMessage))
+                        .failure(PaymentSheetError.linkLookupNotFound)
                     )
                 case .noAvailableLookupParams:
                     completion(
-                        .failure(PaymentSheetError.unknown(debugDescription: "The client secret is missing"))
+                        .failure(PaymentSheetError.missingClientSecret)
```

Comment:
missingClientSecret is very generic.  Is this intentional?  I would have expected we wanted to have link errors start with "link", or something like that?

Reply:
I wasn't the original author here, so I can't comment on this, but the error message is very generic so that's the route I went.